test/e2e/exec_test.go

```diff
@@ -280,7 +274,7 @@ var _ = Describe("Podman exec", func() {
 		exec := podmanTest.Podman([]string{"exec", "-ti", ctrName2, "id"})
 		exec.WaitWithDefaultTimeout()
 		Expect(exec.ExitCode()).To(Equal(0))
-		Expect(strings.Contains(exec.OutputToString(), fmt.Sprintf("%s(%s)", gid, groupName))).To(BeTrue())
+		Expect(exec.OutputToString()).To(ContainSubstring(fmt.Sprintf("%s(%s)", gid, groupName)))
```

Comment:
Should this equal?

Reply:
The output of `id` would at least also have the UID, so we can't just use `Equal` without further changes to the test.

Reply:
I am fine with this, then.